src/phoenix/experimental/evals/long_context_evaluators.py

```diff
@@ -0,0 +1,58 @@
+from typing import List, Optional
+
+from phoenix.experimental.evals import PromptTemplate
+from phoenix.experimental.evals.models import BaseEvalModel
+
+
+class Refiner:
+    def __init__(
+        self,
+        model: BaseEvalModel,
+        initial_prompt_template: PromptTemplate,
+        refine_prompt_template: PromptTemplate,
+        reduce_prompt_template: Optional[PromptTemplate] = None,
+    ) -> None:
+        self._model = model
+        self._initial_prompt_template = initial_prompt_template
+        self._refine_prompt_template = refine_prompt_template
+        self._reduce_prompt_template = reduce_prompt_template
+
+    def evaluate(self, documents: List[str]) -> str:
```

Comment:
```suggestion
    def evaluate(self, documents: List[str]) -> str:
```
clarification: I think your aim here is to make this generic across different larger pieces of data - in that case I guess it might make sense to keep the parameter somewhat generic like "chunk" or "part"?

Reply:
addressed in [637e562](https://github.com/Arize-ai/phoenix/pull/1710/commits/637e56234d6868ff28de933318fdc2083ffd2bbd)